frontend/src/metabase/query_builder/components/template_tags/TagEditorParamParts/FilterWidgetTypeSelect.tsx

```diff
@@ -27,7 +29,7 @@ export function FilterWidgetTypeSelect({
     () =>
       (hasOptions ? options : [{ name: t`None`, type: "none" }]).map(
         option => ({
-          label: option.name,
+          label: option.menuName ?? option.name,
```

Comment:
This is to make filters in native editor work the same as in dashboards: https://github.com/metabase/metabase/blob/9108dd750c5cba9f2da0416f1c27c649f59e8444/frontend/src/metabase/parameters/components/ParameterSettings/ParameterSettings.tsx#L153